### PR TITLE
Apollo time improvement

### PR DESCRIPTION
**Pull request by oded282**
I made some optimizations in Apollo's test, in order to decrease the tests duration.
The following changes are made:

1. I increased replicaConfig.concurrencyLevel to 16.
2. Optimized write operations to work in parallel.
3. Decreased trio.sleep functions.

### CODE REVIEW

File: tests/apollo/util/skvbc_history_tracker.py [hunk 1]
```diff
@@ -904,14 +904,15 @@ def writeset(self, max_size):
...
-            await trio.sleep(.1)
+            async with trio.open_nursery() as nursery:
+                try:
+                    if random.random() < write_weight:
+                        nursery.start_soon(self.send_tracked_write, client, max_size)
+                    else:
+                        nursery.start_soon(self.send_tracked_write, client, max_size)
+                except:
+                    pass
+                await trio.sleep(.01)
```
Comment (teoparvanov) [inline tests/apollo/util/skvbc_history_tracker.py]: I'm not sure we need this sleep here. Have you tried removing it?

File: tests/apollo/test_skvbc_view_change.py [hunk 2]
```diff
@@ -159,8 +159,7 @@ class SkvbcViewChangeTest(unittest.TestCase):
 
             await tracker.tracked_read_your_writes()
 
-            async with trio.open_nursery() as nursery:
-                nursery.start_soon(tracker.run_concurrent_ops, 100)
+            await tracker.run_concurrent_ops(100)
```
Comment (teoparvanov) [inline tests/apollo/test_skvbc_view_change.py]: this "pattern" repeats in several places - can you clean it in the entire codebase?

Comment (oded282) [reply tests/apollo/test_skvbc_view_change.py]: Sure.

File: tests/apollo/util/skvbc_history_tracker.py (hunk 1 above)
Comment (oded282) [reply tests/apollo/util/skvbc_history_tracker.py]: I have tried, it seems to trigger problems in 'test_ro_replica_start_simultaneously' test case.

File: tests/apollo/test_skvbc_ro_replica.py [hunk 3]
```diff
@@ -103,15 +109,18 @@ class SkvbcReadOnlyReplicaTest(unittest.TestCase):
...
+                        try:
+                            key = ['replica', 'Gauges', 'lastExecutedSeqNum']
+                            lastExecutedSeqNum = await bft_network.metrics.get(ro_replica_id, *key)
+                        except KeyError:
+                            continue
+                        else:
+                            # success!
+                            if lastExecutedSeqNum >= 150:
+                                print("Replica" + str(ro_replica_id) + " : lastExecutedSeqNum:" + str(lastExecutedSeqNum))
+                                nursery.cancel_scope.cancel()
```
Comment (teoparvanov) [inline tests/apollo/test_skvbc_ro_replica.py]: those are changes I have made into master - why do we see them here?
Please rebase properly before we can merge this PR...

**Pull request merged**